tests/test_availability_filter.py

```diff
@@ -0,0 +1,24 @@
+from availability_filter import AvailabilityFilter
+from beer import Beer
+from yes_filter import YesFilter
+
+
+def get_test_beer(quantity):
```

Comment:
I like `make_test_beer` better than `get_test_beer`

Can/should this be a pytest fixture?